src/components/journey/Journey.js

```diff
@@ -1,15 +1,180 @@
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
 
-// Component used to display a journey
-const Journey = ({ destination, price }) => (
-    <div>
-        <div>
-            <h1>Destination: {destination}</h1>
-            <p>Price: {price}</p>
-        </div>
-    </div>
-)
+const styles = theme => ({
+    container: {
+        flexDirection: 'column',
+    },
+    bigAvatar: {
+        margin: 10,
+        width: 250,
+        height: 250,
+    },
+    root: {
+        width: '100%',
+        maxWidth: 360,
+        backgroundColor: theme.palette.background.paper,
+        width: 500,
+        margin: '0 auto',
+    },
+    chip: {
+        marginRight: theme.spacing.unit,
+    },
+    section1: {
+        margin: `${theme.spacing.unit * 3}px ${theme.spacing.unit * 2}px`,
+    },
+    section2: {
+        margin: theme.spacing.unit * 2,
+    },
+    section3: {
+        margin: `${theme.spacing.unit * 6}px ${theme.spacing.unit * 2}px ${theme.spacing.unit * 2}px`,
+    },
+});
 
-export default Journey;
+class Journey extends Component {
 
+    constructor(props) {
+        super(props);
+        console.log('hello : ' + props.match.params.id);
+        // Contains the state of the component
+        this.state = {
+            error: null,
+            isLoaded: false,
+            journey: [],
+            journeyId: props.match.params.id
+        }
+
+        console.log('before : ' + props.match.params.id);
+        // It allows for the function to get "this"
+        this.findJourney = this.findJourney.bind(this);
+        console.log('hello : ' + props.match.params.id);
+    };
+
+    componentDidMount() {
+        this.findJourney();
+    }
+
+    /**
+     * Bring a journey from the server (graphql)
+     */
+    getJourney = async (query, variables) => {
+        try {
+            const response = await axios.post('http://localhost:8080/graphql', {
+                query,
+                variables
+            });
+
+            // Log the response so we can look at it in the console
+            console.log('response.data : ' + response.data)
+            // TODO : Gestion des erreurs
+            // Set the data to the state
+            this.setState(() => ({
+                isLoaded: true,
+                firstDisplay: false,
+                journey: response.data.data.findJourneyById
+            }));
+
+        } catch (error) {
+            // If there's an error, set the error to the state
+            console.log('error : ' + error);
+            this.setState(() => ({ error }))
+        }
+    }
+
+    /**
+ * Find a journey
+ */
+    findJourney() {
+        console.log('findJourney');
+        // This is the GraphQL query for findJourney
+        let query = `
+            query findJourney {
+                findJourneyById(id:"`
```

Comment:
please use the interpolation instead of += 
`${var}`

Reply:
Ok